a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -323,13 +321,13 @@ <h4>Instructions</h4>
 
 					<li>
 						<span><b>IF</b> <var>all_necessary_content_textual</var>:</span>
-						<span><b>THEN</b> display <code id="nonvisual-reading-readable">"Readable in read aloud or  dynamic braille"</code>.</span>
+						<span><b>THEN</b> display <code id="nonvisual-reading-readable">"Readable in read aloud or dynamic braille"</code>.</span>
 					</li>
 					<li>
 						<span><b>ELSE IF</b> <var>non_textual_content_images</var> <b>AND</b> <b>NOT</b> <var>textual_alternative_images</var>:</span>
-						<span><b>THEN</b> display <code id="nonvisual-reading-not-fully">"Not fully readable in read aloud or dynamic braille"</code>.</span>
+						<span><b>THEN</b> display <code id="nonvisual-reading-not-fully">"Not fully readable in read aloud or dynamic Braille"</code>.</span>
```

Comment:
These should be lowercase "braille". The usual convention is only uppercase when talking about Louis himself or when braille is the first word in a sentence, or used alone, like in a list, where all the terms are capitalized.
```suggestion
						<span><b>THEN</b> display <code id="nonvisual-reading-not-fully">"Not fully readable in read aloud or dynamic braille"</code>.</span>
```